### ISSUE TITLE
"jQuery Find vs Context, Selector" performance test is misleading

### ISSUE THREAD

**Issue by roberkules** [airbnb/javascript#58]
Hi,

about the "jQuery Find vs Context, Selector" test linked in the performance section.
http://jsperf.com/jquery-find-vs-context-sel/13

I just took the test @ jsPerf, expecting a somewhat similar result as find/context internally end up using the same method (if I remember correctly).

But the test is misleading, comparing 2 uneven tests.

$article = jQuery object containing 10 article nodes
article = only one single article node

So the only things you can compare in the test are:
- "find method (node context)" (48,378) vs "context node" (46,031)
- "find method (jquery context)" (6,679) vs "context jQuery" (7,030)

"context jQuery[0]" is basically the same as "context node"

TL;DR
The linked test is misleading as it's comparing different things, so let's either add a new revision with an accurate comparison or just remove the link?!

BTW, we're about 95% compliant to your style guide, trying now to get to 100%. Great work from you guys!

**Comment by tomekwi** [comment]
@goatslacker @hshoff shouldn’t this be closed? The readme seems to be corrected now.

**Comment by goatslacker** [comment]
Good find
